checker.cpp

```diff
@@ -3,44 +3,63 @@
 #include <unistd.h>
 using namespace std;
 
-int vitalsOk(float temperature, float pulseRate, float spo2) {
-  if(temperature > 102 || temperature < 95) {
-    cout << "Temperature critical!\n";
-    for (int i = 0; i < 6; i++)
+int callSleep()
```

Comment:
Name the function by its purpose. Then its callers will know what to use it for.
This name goes by the implementation (content of the code in the function) - it doesn't offer guidance on its usage